app/forms/mobile_number_form.rb

```diff
@@ -0,0 +1,44 @@
+class MobileNumberForm < Form
+  attribute :mobile_number
+
+  validates :mobile_number,
+    presence: {
+      message: "Enter a mobile number, like 07700 900 982 or +44 7700 900 982"
```

Comment:
Please use the i18n file for error messages for consistency. `i18n_errors_path` should specify where these are.